### ISSUE TITLE
add ability to start a multi-node cluster

### ISSUE THREAD

**Issue by rafaeltuelho** [projectatomic/adb-atomic-developer-bundle#595]
Would be nice have the ability to start an ADB/CDk cluster (eg: one master and one/or more nodes).
Just create a new vm definition on Vagrantfile using a parameter indicating that vm instance is a node (not a master).

**Comment by rafaeltuelho** [comment]
Looking inside the vagrant box vm I've found this script: `/opt/adb/openshift/openshift` which is used to bootstrap/setup the Openshift inside ADB/CDK.

I believe applying some changes to this script is sufficient to decide if it will be an all-in-one, a master or justa  node.

I could not identify where this script come from. I did not find it on this repo.

**Comment by praveenkumar** [comment]
@rafaeltuelho script come from https://github.com/projectatomic/adb-utils

**Comment by LalatenduMohanty** [comment]
Precisely from https://github.com/projectatomic/adb-utils/blob/master/services/openshift/scripts/openshift

**Comment by rafaeltuelho** [comment]
Thanks @LalatenduMohanty, @praveenkumar  !
Do you think this issue should be moved to adb-utils project repo?

**Comment by LalatenduMohanty** [comment]
@rafaeltuelho Yes, It should. You can move it if you are planning to send a PR.
